src/utils.ts

```diff
@@ -1198,4 +1352,786 @@ export class RubberBand {
+    context.lineWidth = this.lineWidth;
+    let color = this.strokeStyle;
+    if (this.mouseStyleON) {color = this.isHover ? this.hoverStyle : this.isClicked ? this.clickedStyle : this.strokeStyle};
+    context.strokeStyle = color;
+    context.fillStyle = color;
+    context.stroke(this.drawPath);
+    context.fill(this.drawPath);
+
+    const canvasHTMatrix = context.getTransform();
+    const pointHTMatrix = canvasHTMatrix.multiply(this.transformMatrix);
+    context.setTransform(pointHTMatrix);
+    this.ticksCoords = this.drawTicks(context, pointHTMatrix, color);
+    context.resetTransform();
+    this.drawName(context, canvasHTMatrix);
+    context.setTransform(canvasHTMatrix);
+    this.drawRubberBand(context);
+  }
+
+  private drawName(context: CanvasRenderingContext2D, canvasHTMatrix: DOMMatrix) {
+    let baseline = ['hanging', 'alphabetic'][Math.sign(1 - Math.sign(this.transformMatrix.f))]
+    let nameCoords = this.end.add(this.origin).divide(2);
+    if (this.isVertical) {nameCoords.x += this.OFFSET_NAME.x ; baseline = ['alphabetic', 'hanging'][Math.sign(1 - Math.sign(this.transformMatrix.e))]}
+    else {nameCoords.y += this.OFFSET_NAME.y}
+    nameCoords.transformSelf(canvasHTMatrix);
+    const orientation = this.isVertical ? -90 : 0;
+    const textName = new newText(this.title, nameCoords, this.drawLength, this.FONT_SIZE, this.FONT, 'center', baseline, 'bold', orientation);
+    textName.draw(context);
+  }
+
+  private drawTicks(context: CanvasRenderingContext2D, pointHTMatrix: DOMMatrix, color: string) {
+    const ticksCoords = [];
+    let count = Math.max(0, this.ticks[0]);
+    this.ticks.forEach((tick, idx) => {
+      if (tick >= this.minValue && tick <= this.maxValue) {
+        let point = this.drawTickPoint(context, tick, this.isVertical, pointHTMatrix, color);
+        ticksCoords.push(point);
+        let text = this.labels[idx]
+        if (this.isDiscrete) {
+          if (count == tick && this.labels[count]) {text = this.labels[count] ; count++}
+          else {text = ''}
+        }
+        this.drawTickText(context, text, point, pointHTMatrix);
+      }
+    })
+    return ticksCoords
+  }
+
+  private drawTickPoint(context: CanvasRenderingContext2D, tick: number, vertical: boolean, HTMatrix: DOMMatrix, color: string): newPoint2D {
+    const markerOrientation = this.isVertical ? 'right' : 'up';
+    const point = new newPoint2D(tick * Number(!vertical), tick * Number(vertical), 10 / Math.abs(HTMatrix.a), 'halfLine', markerOrientation);
+    point.color = color;
+    point.lineWidth /= Math.abs(HTMatrix.a);
+    point.draw(context);
+    return point
+  }
+
+  private drawTickText(context: CanvasRenderingContext2D, text: string, point: newPoint2D, HTMatrix: DOMMatrix): void {
+    const [textOrigin, textAlign, baseline] = this.tickTextPositions(point, HTMatrix);
+    let textWidth = null;
+    if (textAlign == 'left') {textWidth = context.canvas.width - textOrigin.x - 5};
```

Comment:
For one-liner if statements, brackets are not mandatory and often allow a prettier code

Reply:
Removed here, will be removed anywhere in further dev